mail-server/dovecot.nix

```diff
@@ -107,6 +107,8 @@ in
             special_use = \Sent
           }
         }
+
+        sieve = file:${mailDirectory}/%d/%n/sieve;active=${mailDirectory}/%d/%n/.dovecot.sieve
```

Comment:
I went with the approach you mentioned in the other thread: https://github.com/r-raymond/nixos-mailserver/issues/36#issuecomment-345424898

I used dovecot's `%d` (domain) and `%n` (user part of user@domain) variables to store them in the mail dir instead of putting them somewhere in `/var` though.